### PR TITLE
Add numpower to the GitHub workflow pipeline

### PR DESCRIPTION
**Pull request by tanmayk**
https://github.com/RubixML/ML/issues/325

### CODE REVIEW

**Comment by tanmayk** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by andrewdalpino** [comment]
Nice, @tanmayk thank you! I'm curious, for the OpenBLAS dependency, could we install that from a repo such as the Debian one like we do with the Tensor extension?

https://github.com/RubixML/Tensor/blob/master/.github/workflows/ci-ext.yml#L62

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,6 +23,23 @@ jobs:
           extensions: svm, mbstring, gd, fileinfo, swoole
           ini-values: memory_limit=-1
 
+      - name: Install OpenBLA
```
Comment (andrewdalpino) [inline .github/workflows/ci.yml]: "Install OpenBLAS" - add an 'S' here

Comment (tanmayk) [reply .github/workflows/ci.yml]: Fixed. Thanks.

**Comment by tanmayk** [comment]
@andrewdalpino Thank you for the review. Yeah you are right. I've updated the PR accordingly.

**Comment by tanmayk** [comment]
@andrewdalpino Any thoughts? If this gets merged, all [these](https://github.com/RubixML/ML/issues?q=is%3Aissue+is%3Aopen+label%3Aoptimization+-label%3A%22help+wanted%22+) issues can be tested with pipeline.

**Pull request merged**